code/modules/mob/living/carbon/species.dm

```diff
@@ -250,41 +251,34 @@
 				H << "\red You feel your face freezing and an icicle forming in your lungs!"
 		else if(breath.temperature > heat_level_1)
 			if(prob(20))
-				if(H.dna.mutantrace == "slime")
-					H << "\red You feel supercharged by the extreme heat!"
-				else
-					H << "\red You feel your face burning and a searing heat in your lungs!"
-
-		if(H.dna.mutantrace == "slime")
```

Comment:
it's just a flavortext message---they can't even see it because they don't breathe; the cold = damaging them is also rather pointless again---'cause they don't breathe.